### PR TITLE
docs(techniques): renamed fastify plugins

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: 
https://github.com/nestjs/nest/issues/9542

## What is the new behavior?
Renamed fastify plugins:
- `@fastify/cookie`
- `@fastify/compress`
- `@fastify/swagger`
- `@fastify/helmet`
- `@fastify/static`

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/techniques/compression.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ app.use(compression());
 If using the `FastifyAdapter`, you'll want to use [fastify-compress](https://github.com/fastify/fastify-compress):
 
 ```bash
-$ npm i --save fastify-compress
+$ npm i --save @fastify-compress
```
Comment (micalevisk) [inline content/techniques/compression.md:30]:
```suggestion
$ npm i --save @fastify/compress
```

File: content/techniques/compression.md [hunk 2]
```diff
@@ -27,13 +27,13 @@ app.use(compression());
...
 ```
 
 Once the installation is complete, apply the fastify-compress middleware as global middleware.
 
 ```typescript
-import compression from 'fastify-compress';
+import compression from '@fastify-compress';
```
Comment (micalevisk) [inline content/techniques/compression.md:36]:
```suggestion
import compression from '@fastify/compress';
```

File: content/techniques/compression.md [hunk 3]
```diff
@@ -46,4 +46,4 @@ To specify encodings, provide a second argument to `app.register`:
 await app.register(compression, { encodings: ['gzip', 'deflate'] });
 ```
 
-The above tells `fastify-compress` to only use gzip and deflate encodings, preferring gzip if the client supports both.
+The above tells `@fastify-compress` to only use gzip and deflate encodings, preferring gzip if the client supports both.
```
Comment (micalevisk) [inline content/techniques/compression.md:49]:
```suggestion
The above tells `@fastify/compress` to only use gzip and deflate encodings, preferring gzip if the client supports both.
```

File: content/techniques/cookies.md [hunk 4]
```diff
@@ -58,13 +58,13 @@ findAll(@Res({ passthrough: true }) response: Response) {
 First install the required package:
 
 ```shell
-$ npm i fastify-cookie
+$ npm i @fastify-cookie
```
Comment (micalevisk) [inline content/techniques/cookies.md:61]:
```suggestion
$ npm i @fastify/cookie
```

File: content/techniques/cookies.md [hunk 5]
```diff
@@ -58,13 +58,13 @@ findAll(@Res({ passthrough: true }) response: Response) {
...
 ```
 
-Once the installation is complete, register the `fastify-cookie` plugin:
+Once the installation is complete, register the `@fastify-cookie` plugin:
```
Comment (micalevisk) [inline content/techniques/cookies.md:64]:
```suggestion
Once the installation is complete, register the `@fastify/cookie` plugin:
```

File: content/techniques/cookies.md [hunk 6]
```diff
@@ -58,13 +58,13 @@ findAll(@Res({ passthrough: true }) response: Response) {
...
 
 ```typescript
-import fastifyCookie from 'fastify-cookie';
+import fastifyCookie from '@fastify-cookie';
```
Comment (micalevisk) [inline content/techniques/cookies.md:67]:
```suggestion
import fastifyCookie from '@fastify/cookie';
```

**Review by micalevisk — CHANGES_REQUESTED** [review]

**Review by micalevisk — APPROVED** [review]

**Comment by Jared-Pro** [comment]
This PR should be merged. I installed multiple deprecated fastify packages due to the docs being outdated.

**Comment by shunkica** [comment]
Why is it taking so long for such a simple change? I just wasted an hour because the docs said to use `fastify-cookie` but that package produced errors for me.

**Comment by kamilmysliwiec** [comment]
Accidentally merged this PR @Tony133. Can you recreate it?

**Comment by Tony133** [comment]
Yes sure, I'll recreate the PR 👍

**Pull request merged**